src/Chemfiles.jl

```diff
@@ -128,7 +128,7 @@ module Chemfiles
     <http://chemfiles.readthedocs.io/en/latest/selections.html>`_ for more
```

Comment:
This URL is out of date =) Could you change it to http://chemfiles.org/chemfiles/latest/selections.html?